eo-runtime/src/main/java/org/eolang/AtLocated.java

```diff
@@ -65,7 +65,12 @@ public String toString() {
 
     @Override
     public String φTerm() {
-        return this.origin.φTerm();
+        return String.format(
+            "<%d:%d>%s",
```

Comment:
@mximp this is not really phi-expression :)

Reply:
> @mximp this is not really phi-expression :)
@yegor256 I understand - but phi-calculus has no notion of line and position. 
Here it's more like "debug symbols" within phi-expression.

Reply:
@yegor256 do you think we need exclude non-phi info from this method?

Reply:
@mximp yes, this is not the right place to keep this info. Maybe let's add one more method to `Phi` interface called `location()`? It will be implemented only by `PhLocated`, while all others will return `???`. Just an idea.